app/controllers/candidate_interface/personal_details_controller.rb

```diff
@@ -17,6 +17,13 @@ def update
       end
     end
 
+    def show
+      personal_details_form = PersonalDetailsForm.build_from_application(
+        current_candidate.current_application,
+      )
+      @personal_details_review = PersonalDetailsReviewPresenter.new(personal_details_form)
+    end
```

Comment:
as discussed, this should be refactored, we should not create a form to get a present.
but no time for now, I added it in a list of tech debt (I'll create a trello soon)